pom.xml

```
@@ -7,6 +7,70 @@
     <groupId>fr.esir.jxc</groupId>
     <artifactId>user_event_handler</artifactId>
     <version>1.0-SNAPSHOT</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>6</source>
+                    <target>6</target>
```

```suggestion
                    <source>8</source>
                    <target>8</target>
```

On travaille avec Java 8 et pas 6 ;)